docs/api.rst

```diff
@@ -58,3 +58,13 @@ Kymotracking
     track_greedy
     track_lines
     filter_lines
+
+Notebook widgets
+----------------
+
+.. autosummary::
+    :toctree: _api
+
+    FdRangeSelector
+    nb_widgets.fd_selector.SliceRangeSelectorWidget
+    nb_widgets.fd_selector.FdRangeSelectorWidget
```

Comment:
nitpick: We can lose the last `Widget` in the name

Reply:
I can indeed kill `Widget` for `SliceRangeSelector`. Will do! 👍